runtime/src/lib.rs

```diff
@@ -776,7 +776,7 @@ impl pallet_remote_asset_manager::Config for Runtime {
     type XcmExecutor = XcmExecutor<XcmConfig>;
     type XcmAssets = xcm_assets::XcmAssetExecutor<XcmAssetConfig>;
     // Using root as the admin origin for now
-    type AdminOrigin = frame_system::EnsureRoot<AccountId>;
+    type AdminOrigin = frame_system::EnsureSigned<AccountId>;
```

Comment:
`EnsureSigned` for completing the tests first, we need an origin checklist before connecting to Polkadot or rococo TestNet after the testing, lots of the calls don't support `CommiteMemer` or `Root`, because they calls a `let caller = ensure_signed(origin)?` which will unwrap the origin and expect an accountId